src/cpu/aarch64/acl_utils.cpp

```diff
@@ -112,6 +117,34 @@ void acl_thread_bind() {
         arm_compute::Scheduler::get().set_num_threads(max_threads);
     });
 }
+#endif
+
+#if DNNL_CPU_THREADING_RUNTIME == DNNL_RUNTIME_THREADPOOL
+void acl_set_custom_scheduler() {
```

Comment:
(minor) seems like there are quite a few threading related functions, which can be moved into something like `src/cpu/aarch64/acl_thread.{hpp/cpp}` similar to https://github.com/oneapi-src/oneDNN/blob/master/src/common/dnnl_thread.cpp

Reply:
Thanks for the suggestion. I have made the change.

Reply:
Thanks!